index.html

```diff
@@ -9,5 +9,12 @@
   </head>
   <body>
     <header>QuizApp for WebDevs</header>
+    <main>
+      <section class="card1">
```

Comment:
I guess you will want to style all card the same, so `card1` is not a proper naming. You could choose `card` instead.